lms/js_config_types.py

```diff
@@ -36,7 +36,12 @@ class APICourse(TypedDict):
 
 
 class APIStudent(TypedDict):
-    id: str
+    h_userid: str
```

Comment:
I reckon this is the right name, otherwise it could become confusing with other IDs.

When we filter by it we should also match this naming on the query parameter.